pulsar-broker/src/test/java/org/apache/pulsar/broker/resourcegroup/ResourceQuotaCalculatorImplTest.java

```diff
@@ -111,5 +111,15 @@ public void testRQCalcGlobUsedZeroTest() throws PulsarAdminException {
         Assert.assertEquals(newQuota, config);
     }
 
+    @Test
+    public void testNeedToReportLocalUsage() {
```

Comment:
Thanks for the fix. Before this fix, this test will failed, right?

Reply:
Yes, before this fix,  result of `(diff / currentBytesUsed) * 100` is 0 if diff is 6 and currentBytesUsed is 100